### PR TITLE
Update TheTheme

### PR DESCRIPTION
**Pull request by TFleury**
Allow easy "brand" overriding and clean CSS.

### CODE REVIEW

**Comment by hishamco** [comment]
I think I need to test this one after merge #4069

**Comment by agriffard** [comment]
Please solve the conflicts @TFleury

**Comment by TFleury** [comment]
> Please solve the conflicts @TFleury

Done

File: src/OrchardCore.Themes/TheTheme/wwwroot/css/sticky-footer-navbar.css [hunk 1]
```diff
@@ -27,13 +27,4 @@ footer {
...
-
-footer > .container {
-    padding-right: 15px;
-    padding-left: 15px;
-}
-
-code {
-    font-size: 80%;
-}
+}
```
Comment (Skrypt) [inline src/OrchardCore.Themes/TheTheme/wwwroot/css/sticky-footer-navbar.css:30]: eof

**Review by Skrypt — APPROVED** [review]
LGTM apart the small eof

**Pull request merged**